mypy.ini

```diff
@@ -2,4 +2,4 @@
 ignore_missing_imports = True
 
 ; FIXME: #4052 fix mypy errors in the exclude directories and remove them below
-exclude = (data_structures|dynamic_programming|graphs|maths|matrix|other|project_euler|searches|strings*)/$
+exclude = (data_structures|graphs|maths|matrix|other|project_euler|searches|strings*)/$
```

Comment:
Please remove this change as the directory is not completely typed.

Reply:
If we want to make sure that there are no untyped functions, we could use the additional parameter `--disallow-incomplete-defs`, see https://mypy.readthedocs.io/en/stable/command_line.html#untyped-definitions-and-calls.

Reply:
Thank you looking into it. We are going to make `mypy` stricter, but not now.